### ISSUE TITLE
QA Ph3_Accessibility: Schedule - Unable to access tabs following voice over instructions 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#664]
With voice over enabled, user is unable to access the Schedule tabs following the instructions given by voice over.  

Steps: 
1. Enable voice over in OS settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Select Schedule 
4. Use the tab key to navigate to the Schedule tabs (Agenda, May 18, etc.) 
5. Use the right or left arrow button to navigate between tabs 
6. Voice over instructions will read "to select this option, press Control-Option-Space" 
7. Follow cues and select control-option-space 
8. Voice over cue changes to "press my schedule" 
9. Tap enter 
10. Note no response - use cannot access tabs 

Chrome 49 Mac OS X 10.11

![gio_664](https://cloud.githubusercontent.com/assets/15128124/14469474/a5e9a206-0099-11e6-85b9-61c8f5983be3.png)
